package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "estree",
+  "version": "1.0.0-pre",
+  "description": "ECMAScript syntax representation format",
+  "main": "spec.md",
```

Comment:
Node.js uses `mail` for `require` calls, Markdown in `main` wouldn't make sense and just confuses IMO.